quarto/src/Entity/Game.php

```diff
@@ -0,0 +1,69 @@
+<?php
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass="App\Repository\GameRepository")
+ * @ORM\Table
+ */
+class Game {
+    /**
+     * @ORM\Column(type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id_game;
```

Comment:
In PHP, we use camelCase and not snake_case

Reply:
Actually, it is not recommanded to have upper case chars in PostgreSQL names, so properties are only in lower case in Entity to ensure compatibility